database.c

```diff
@@ -70,7 +70,6 @@ bool dbquery(const char *format, ...)
 	if(query == NULL)
 	{
 		logg("Memory allocation failed in dbquery()");
-		va_end(args);
```

Comment:
This is already called before in line [68](https://github.com/pi-hole/FTL/pull/117/files#diff-b6b5457d88ce99d5834dbf78cd082140L68). No need to do it again.